test/configs/edge-ostree-pull-user.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "edge-ostree-pull-user",
+  "blueprint": {
+    "customizations": {
+      "user": [
+        {
+          "name": "osbuild",
+          "key": "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIPDyeUyVWvuSSOdrikiDWgNoCz0oYP4Ir68jJecn+XYc github.com/osbuild/images"
+        }
+      ]
```

Comment:
Looking at the CI results, the user does not seem to be in any of the sudoers group and I'm not sure if password-less sudo is enabled at all in the image. This is what `cloud-init` does on first boot, so we need to do all of this on our own :disappointed:

Reply:
Yeah, that's what I meant by "kinda" :laughing: 

We don't have a clean way to create passwordless sudo.  I think there are a couple of options here:
1. Log in as root to run the script.
2. Add the user to the appropriate groups so it can run `systemctl` commands without root.

I'm going to try 2. first and if we ever need root to run things for the test (which I imagine we eventually will) we can figure it out then.

Reply:
we could use the files customization:

```toml
[[customizations.files]]  
path = "/etc/sudoers.d/wheel-passwordless-sudo"  
mode = "0400"  
data = """  
%wheel ALL=(ALL) NOPASSWD: ALL  
"""
```

Reply:
Of course, file customizations aren't enabled for `edge-ami`

Reply:
Did a quick manual test and I think simply adding the user to the wheel group might be enough.  Let's find out.

Reply:
Silly me.  I was running `sudo systemctl is-system-running` and as far as I can tell, that doesn't need `sudo`.